c/rust-tests/src/tests/smt.rs

```diff
@@ -156,7 +157,7 @@ impl Hasher for CkbBlake2bHasher {
 
 pub type CkbSMT = SparseMerkleTree<CkbBlake2bHasher, H256, DefaultStore<H256>>;
 
-pub fn new_ckb_smt(pairs: Vec<(H256, H256)>) -> CkbSMT {
+pub fn _new_ckb_smt(pairs: Vec<(H256, H256)>) -> CkbSMT {
```

Comment:
Why a public function start with `_` ? Even it's a private function, the absent of keyword `pub` already means it's a private function.

Reply:
Try uncomment the `test_fixtures()` function. It's a historical issue yet to be fixed.

Reply:
OK